src/utils/currencies.ts

```diff
@@ -287,6 +594,36 @@ export const FIAT_OPTIONS: Currency[] = [
         hasSymbol: "₫",
         maxFractionalDigits: 0
     },
+    {
+        label: "Central African CFA Franc BEAC XAF",
+        value: "XAF",
+        hasSymbol: "CFA Franc BEAC",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Silver Ounce XAG",
+        value: "XAG",
+        hasSymbol: "Oz",
+        maxFractionalDigits: 6
+    },
+    {
+        label: "Gold Ounce XAU",
+        value: "XAU",
+        hasSymbol: "Oz",
+        maxFractionalDigits: 6
+    },
+    {
+        label: "CFA Franc XOF",
+        value: "XOF",
+        hasSymbol: "Franc",
+        maxFractionalDigits: 2
+    },
```

Comment:
And for the West African Franc
```suggestion
    {
        label: "West African CFA Franc XOF",
        value: "XOF",
        hasSymbol: "F.CFA",
        maxFractionalDigits: 2
    },
```

Reply:
sad to say it does have a single character sure, but I think either F.CFA or FCFA is fine, but I will prefer the latter @benalleng 
cc: @benthecarman